src/tics.jl

```diff
@@ -0,0 +1,114 @@
+function _tics{T<:AbstractFloat}(radec_min::T, radec_max::T, numx::T, ticsize::T, ra::Bool)
+
+    numtics = numx/ticsize
+    if ra
+        mul = 4.0
```

Comment:
Apologies, I didn't realise the significance of T(number) earlier.
Added T() to all assignment areas.